src/test-data/OtherData.js

```diff
@@ -0,0 +1,94 @@
+import { backMatterTestData } from "./BackMatterData";
+
+const adds = [
```

Comment:
This seems specific to modifications, should it be moved there?

Reply:
Are you referring to the adds const?

Reply:
adds is only used in that file and isn't referenced anywhere else, that is why I placed it in OtherData.js (or whatever we rename that file to)

Reply:
For #124 having the `modify` including the `adds` portion of a profile is used in places other than just `Profile` (`SSP` & `Component Definition`. Maybe having a `ModificationData` file makes sense? I am not sure if that is exactly what you mean @rgauss but it could prevent some duplication later.

Reply:
It is related to modifications, but the only place it's used is the profile test. 
I think it'd technically be cleanest to modify the profile test to use the existing
test data in `test-data/ModificationsData` but that might be more trouble than it's worth.

Reply:
I moved this to a more accurately named 'ProfileData.js` - hope that makes this more readable.